### DISCUSSION TITLE
Consider errors as warnings

### DISCUSSION THREAD

**Discussion by jmabra01** [FirelyTeam/firely-net-sdk#1675] [Q&A] [answered]
I am using this library to validate my resources and I get an error on the extenisons - {Text="Unable to resolve reference to profile 'http://XXX/patient-code'"} this is expected in my case as the extensions haven't been published yet. Is there a way to treat these as just warnings. and not error to be able to validate the bundle.

**Comment by mmsmits** [comment, marked as answer]
Thanks for starting this discussion! There currently isn't a way to treat specific errors as warnings yourself. 
However, we are in the process of aligning the output of different validators out there (Java and .NET), and will also be discussing which outputs are warnings, and which are errors.
When we do this, we will definitely  take your feedback about this specific error (not able to resolve extensions) into account.

If you encounter similar problems, please add those to this discussion.
